features/step-easing.yml

```
@@ -0,0 +1,7 @@
+name: steps() easing
+description: The `steps()` easing function divides an animation or transition into equidistant steps. The `step-start` and `step-end` CSS properties are presets for specific scenarios.
```

```suggestion
The `steps()` easing function breaks an animation or transition into a number of equally spaced intervals, creating a segmented or jumpy effect. The `step-start` and `step-end` CSS properties are presets with a single step.
```

Attempt at making the definition non-circular, not sure I love it.

---

How about:
> The `steps()` easing function divides an animation or transition into equidistant intervals, jumping from value to value. The `step-start` and `step-end` CSS properties are presets with a single step.

---

Looks good to me!